CRM/Event/Form/ManageEvent/TabHeader.php

```diff
@@ -66,7 +66,14 @@ public static function process(&$form) {
     $tabs = [];
     $tabs['settings'] = ['title' => ts('Info and Settings'), 'class' => 'ajaxForm livePage'] + $default;
     $tabs['location'] = ['title' => ts('Event Location')] + $default;
-    $tabs['fee'] = ['title' => ts('Fees')] + $default;
+    // Check to see if CiviContribute is an enabled component.
+    $components = \Civi\Api4\Setting::get()
```

Comment:
The helper `Civi::settings()->get('enable_components')` is a little pithier (we also support that for extensions/outside core)